### DISCUSSION TITLE
Ezra Bible App - Year 2022 in review!

### DISCUSSION THREAD

**Discussion by tobias-klein** [ezra-bible-app/ezra-bible-app#859] [General]
What happened in 2022 for Ezra Bible App? :)

* Five releases.
* Support for Android mobile with optimizations for mobile layout / UX.
* 95 features/enhancements/issues closed!
* The JavaScript code base grew from 19.8K lines of code to 24.8K lines of code.
* Major features added:
  * CSV export of all user data (tags, notes).
  * Support deuterocanonical / apocryphal books.
  * Tags can now be categorized / grouped in tag groups.
  * Support for Android mobile!
  * Dropbox Sync functionality.
  * Module update functionality.

As the maintainer of Ezra Bible App, I am very thankful for what could be done in the last year. A special thank you to all translators who have been consistently contributing! (@lemtom @augustin-colesnic @MartinIIOT @reyespinosa1996 @zhuiks) 

Perspectively it would be nice to have additional contributors in the area of software development as well as contributors in the area of marketing / user guidance, etc.

May this all be to God's glory and to the benefit of our user's Bible study!

**Comment by reyespinosa1996** [comment]
First thank you @tobias-klein for your hard work, Ezra Bible App wouldn't be the really great software it is today if it wasn't for you.  As you may have seen, I'm not a GitHub person myself. But, I do have some experiences in community management, user guidance and costumers support. We should talk about open some more ways of communicating with the community, maybe a Telegram group or a Discord server will be a great way to engage with folks out there and spread the word about Ezra Bible App. Best regards. 

PS: I'm sorry I just read this today. Hope to get over here more often :)
